packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -53,10 +53,10 @@ abstract final class PointerEventConverter {
   /// Expand the given packet of pointer data into a sequence of framework
   /// pointer events.
   ///
-  /// The `devicePixelRatio` argument (usually given the value from
-  /// [dart:ui.FlutterView.devicePixelRatio]) is used to convert the incoming data
-  /// from physical coordinates to logical pixels. See the discussion at
-  /// [PointerEvent] for more details on the [PointerEvent] coordinate space.
+  /// The [DevicePixelRatioGetter] is used to obtain the device pixel ratio for
```

Comment:
I think our convention is to use the variable name, not the type name, so it should be 
```suggestion
  /// The `devicePixelRatioForView` is used to obtain the device pixel ratio for
```